src/main/java/frc/robot/subsystems/Leds.java

```diff
@@ -0,0 +1,248 @@
+package frc.robot.subsystems;
+
+import com.chopshop166.chopshoplib.outputs.DigitalOutputDutyCycle;
+import com.mach.LightDrive.*;
+
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.InstantCommand;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import frc.robot.RobotMap;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+
+public class Leds extends Subsystem {
+    LightDriveCAN ldrive_can;
+    Color seafoam = new Color(150, 238, 150);
+    Color amber = new Color(255, 191, 0);
+    Color harlequin = new Color(43, 255, 0);
+    Color fuschia = new Color(255, 0, 255);
+
+    public Leds() { // NOPMD
+        super();
+        // Take values that the subsystem needs from the map, and store them in the
+        // class
+        
+        ldrive_can = new LightDriveCAN();
+        ldrive_can.SetColor(1, Color.BLUE, 1.0);
+        ldrive_can.SetColor(2, Color.BLUE, 1.0);
+
+        ldrive_can.Update();
+    }
+
+    @Override
+    public void initDefaultCommand() {
+        setDefaultCommand(setTeamColor());
+    }
+
+    private boolean isBlueTeam() {
+        Alliance team = DriverStation.getInstance().getAlliance();
+        if (team == DriverStation.Alliance.Blue) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public Command setTeamColor() {
+        // The command is named "Sample Command" and requires this subsystem.
+        return new Command("Set team color", this) {
+
+            protected void initialize() {
+                // Called just before this Command runs the first time
+                if (isBlueTeam()) {
+                    ldrive_can.SetColor(1, Color.BLUE, 1.0);
+                    ldrive_can.SetColor(2, Color.BLUE, 1.0);
+                    ldrive_can.SetColor(3, Color.BLUE, 1.0);
+                    ldrive_can.SetColor(4, Color.BLUE, 1.0);
+
+                } else {
+                    ldrive_can.SetColor(1, Color.RED, 1.0);
+                    ldrive_can.SetColor(2, Color.RED, 1.0);
+                }
```

Comment:
If `SetColor(3, ...)` isn't called, maybe leave a comment for why blue needs it and not red?